### DISCUSSION TITLE
wrong github repo

### DISCUSSION THREAD

**Discussion by pfeerick** [headtracker/HeadTracker#73] [Q&A]
Ignore me... should have looked which github repo I was on... 🤦 And I don't seem to be able to delete my own post.

**Comment by dlktdr** [comment]
Lol.. Just looked at this. I'm like am I on the wrong discord? Why am I GitHub updates showing Edge stuff.. :)

**Reply by pfeerick** [reply]
Don't you worry, If I get involved in the headtracker that could very well happen! 😈 I have a pair of Arduino Nano 33 BLEs that arrived yesterday, next up will to be to figure out what tilt mechanism (or possibly pan and tilt) I want to use.... might end up 3D printing one...

**Reply by dlktdr** [reply]
Nice!.. Welcome to the club. 

I liked this one. had to tweak it for my servo tho. https://www.thingiverse.com/thing:2779954 depending where you mount it was fairly aerodynamic and stable because the two halves contact on the pan.

There are a few on the headtracker discord under #pan tilt mounts too.. Among the others on thingiverse.

**Reply by pfeerick** [reply]
Oh, nice... thanks! I was looking at a very nice commercial one... half gears thingy... I thought it quite simple and elegant... see what happens anyway.

**Reply by dlktdr** [reply]
If your thinking about the BAG by Motionsic. it is very nice.. he put really nice servos on it and they are 180deg out of the box. Defiantly recommended.

So many projects... time to put in a bit more effort on model select. ;)
